src/qibojit/backends/gpu.py

```diff
@@ -15,13 +15,16 @@ class CupyBackend(NumbaBackend):  # pragma: no cover
 
     def __init__(self):
         NumpyBackend.__init__(self)
-        import os
 
         import cupy as cp  # pylint: disable=import-error
         import cupy_backends  # pylint: disable=import-error
 
         self.name = "qibojit"
         self.platform = "cupy"
+        self.versions[
+            "cupy" : cp.__version__,
+        ]
```

Comment:
There is a syntax error here
```suggestion
        self.versions["cupy"] = cp.__version__
```
It is not captured by CI because it does not use the GPU backends.